model/riscv_sys_control.sail

```diff
@@ -100,6 +100,9 @@ function is_CSR_defined (csr : csreg, p : Privilege) -> bool =
     0x343 => p == Machine, // mtval
     0x344 => p == Machine, // mip
 
+    0x747 => p == Machine, // mseccfg
+    0x757 => p == Machine & (sizeof(xlen) == 32), // mseccfgh
+
```

Comment:
I don't think that is strictly correct. You can have `mseccfg` without PMPs. Really we need a `sys_has_mseccfg()` flag or something like that, because it is optional. (As far as I remember anyway.)

Reply:
I think you're right. Either way this code needs to be altered before we can merge.